RELEASE/scripts/autoscend/auto_quest_level_9.ash

```diff
@@ -657,41 +657,18 @@ boolean L9_twinPeak()
 	{
 		return false;
 	}
+	auto_log_info("Twin Peak", "blue");
 
-	set_property("choiceAdventure609", "1");
-	if(attemptNum == 1)
-	{
-		set_property("choiceAdventure606", "1");
-		set_property("choiceAdventure607", "1");
-	}
-	else if(attemptNum == 2)
-	{
-		set_property("choiceAdventure606", "2");
-		set_property("choiceAdventure608", "1");
-	}
-	else if(attemptNum == 3)
-	{
-		set_property("choiceAdventure606", "3");
-		set_property("choiceAdventure609", "1");
-		set_property("choiceAdventure616", "1");
-	}
-	else if(attemptNum == 4)
-	{
-		set_property("choiceAdventure606", "4");
-		set_property("choiceAdventure610", "1");
-		set_property("choiceAdventure1056", "1");
-	}
-
-	int trimmers = item_amount($item[Rusty Hedge Trimmers]);
-	if(item_amount($item[Rusty Hedge Trimmers]) > 0)
+	int starting_trimmers = item_amount($item[Rusty Hedge Trimmers]);
+	if(starting_trimmers > 0)
 	{
 		use(1, $item[rusty hedge trimmers]);
 		cli_execute("refresh inv");
-		if(item_amount($item[rusty hedge trimmers]) == trimmers)
+		if(item_amount($item[rusty hedge trimmers]) == starting_trimmers)
 		{
 			abort("Tried using a rusty hedge trimmer but that didn't seem to work");
 		}
```

Comment:
(optional) should not be necessary to do this. `use(1, $item[rusty hedge trimmers])` "returns true if the item is used or false if it fails to do so" according to the [kolmafia wiki](https://wiki.kolmafia.us/index.php?title=Use) so actually checking the return value is all that should be required here and the `starting_trimmers` value can be removed.

Reply:
Oh, that is a good point
Something for a followup PR then.
As it looks like it was meant to fix some old mafia bug that probably does not apply anymore. But just to be sure I would like to test whatever changes I make here. I tested the rest of the changes thoroughly already so I would like to merge them now.
Thank you for pointing this out.

Reply:
Ignore it, I'll clean it up when I do the choiceAdv stuff.